### ISSUE TITLE
[Question] PRs being opened and then autoclosed

### ISSUE THREAD

**Issue by djmitche** [renovatebot/config-help#397]
See
* https://github.com/taskcluster/tc-admin/pull/38
* https://github.com/taskcluster/tc-admin/pull/39
* https://github.com/taskcluster/tc-admin/pull/40
* https://github.com/taskcluster/tc-admin/pull/41
* https://github.com/taskcluster/tc-admin/pull/42

### Which Renovate are you using? CLI, App, or Pro

App

### Which platform are you using? GitHub, GitLab, Bitbucket Azure DevOps

GitHub

### Have you checked the logs? Don't forget to include them if relevant

https://app.renovatebot.com/dashboard#github/taskcluster/tc-admin/106993579

`WARN: Failed to read setup.py file(packageFile="setup.py")` due to what looks like a docker failure?  The error message says success so I'm not sure what to make of it.

### What is your question?

More of a bug report: if docker fails in trying to run setup.py, I would think the run should fail, vs. deciding there are no updates to do and autoclosing PRs. But perhaps I have something wrong in the config for this repo?

**Comment by rarkins** [comment]
Yes agreed, it should ideally abort the run if there’s any transient error reading the setup.py. The challenge is always working out what’s transient or not. I’ll take a look here. 

Additionally we need to work out why that’s happening too.

**Comment by djmitche** [comment]
It's particularly annoying since I can't re-open the PRs even if they were valid!

**Comment by rarkins** [comment]
Because the branches are deleted?

**Comment by djmitche** [comment]
Yes.  For a few things that renovate doesn't know how to update, I pull the branch, twiddle some knobs, re-push, and then check back later and merge if tests are green.  But when renovate autocloses, it deletes my changes (I still have them locally, and anyway they're trivial, it's just a nuisance).
